src/main/js/field/TableDisplay.js

```diff
@@ -0,0 +1,223 @@
+                    <FontAwesomeIcon icon="exclamation-triangle" className="alert-icon" size="lg" />{cell}
+                </span>
+                : cell;
+
+            if (cell) {
+                return <div title={cell.toString()}> {cellContent} </div>;
+            }
+            return <div> {cellContent} </div>;
+        }
+
+        return this.props.columns.map(column => (
+            <TableHeaderColumn key={column.header} dataField={column.header} isKey={column.isKey} dataSort columnClassName="tableCell" tdStyle={{ whiteSpace: 'normal' }}
+                               dataFormat={assignDataFormat}>{column.headerLabel}</TableHeaderColumn>
+        ));
+    }
+
+    createButtonGroup(buttons) {
+        const classes = 'btn btn-md btn-info react-bs-table-add-btn tableButton';
+        const insertOnClick = buttons.insertBtn ? buttons.insertBtn.props.onClick : null;
+        const deleteOnClick = buttons.deleteBtn ? buttons.deleteBtn.props.onClick : null;
+        const refreshButton = !this.props.autoRefresh &&
+            (<button type="button" tabUserTable={0} className={classes} onClick={this.props.retrieveData}>
+                <FontAwesomeIcon icon="sync" className="alert-icon" size="lg" />Refresh
+            </button>);
+        return (
+            <div>
+                {buttons.insertBtn
+                && <InsertButton className="addJobButton btn-md" onClick={() => {
+                    insertOnClick();
+                    this.flipShowSwitch()
+                }}>
+                    <FontAwesomeIcon icon="plus" className="alert-icon" size="lg" />
+                    New
+                </InsertButton>
+                }
+                {buttons.deleteBtn
+                && <DeleteButton className="deleteJobButton btn-md" onClick={deleteOnClick}>
+                    <FontAwesomeIcon icon="trash" className="alert-icon" size="lg" />
+                    Delete
+                </DeleteButton>
+                }
+                {refreshButton}
+            </div>
+        );
+    }
+
+    handleClose() {
+        this.refs.table.cleanSelected();
+        this.flipShowSwitch();
+    }
+
+    handleSubmit() {
+        this.handleClose();
+        this.props.onConfigSave();
+    }
+
+    flipShowSwitch() {
+        this.setState({
+            show: !this.state.show
+        });
+    }
+
+    createInsertModal(onModalClose) {
+        return (
+            <Modal size="lg" show={this.state.show} onHide={() => {
+                this.handleClose();
+                onModalClose()
+            }}>
+                <Modal.Header closeButton>
+                    <Modal.Title>{this.props.modalTitle}</Modal.Title>
+                </Modal.Header>
+                <Modal.Body>
+                    <form className="form-horizontal" onSubmit={this.handleSubmit} noValidate>
+                        {this.props.children}
+                        <ConfigButtons
+                            cancelId="job-cancel"
```

Comment:
I think the cancelId and showId need to change to be unique.  Looks like a copy paste error.